web/package.json

```diff
@@ -6,21 +6,25 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "format": "prettier --write .",
+    "format-check": "prettier --check ."
```

Comment:
![Van prettier](https://github.com/user-attachments/assets/38332d1c-58d9-4cbf-852e-38268c9a46aa)

I know our `.prettierignore` file should ignore directories like `node_modules` but it's best you specify why prettier should `write` to

And plus i said a `format:check` script not a `format-check` script

```suggestion
"format": "prettier --write ./src",
"format:check": "prettier --check ./src"
```